variables.tf

```diff
@@ -18,8 +31,9 @@ variable "region" {
 }
 
 variable "prefix" {
-  description = "The prefix that you would like to append to your resources. Explicitly set to null if you do not wish to use a prefix."
+  description = "The value that you would like to prefix to the name of the resources provisioned by this module. Explicitly set to null if you do not wish to use a prefix."
   type        = string
+  default     = null
```

Comment:
thoughts on maybe adding a default prefix value here instead of `null` to be consistent with `name` variable? Or should we default `name` to null? cc @toddgiguere

Reply:
we will need some more code modification to support adding default value for `prefix`, while adding a default value for `name` works fine.
![Screenshot 2023-11-15 at 6 17 27 PM](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc/assets/122446118/7775cc72-81e4-4081-8b98-618b878448f0)

Reply:
I was suggesting we either set `name` and `prefix` to `null`, or give both default string values

Reply:
If we are to set `name` and `prefix` to `null`, we will need to update the naming logic everywhere for the existing-vpc example to work.